digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculator.scala

```diff
@@ -125,15 +125,22 @@ object ProjectDeltaCalculator {
 
     val oppositeStatusNotChanged = if (oppositeOriginalAddressLinks.size == 2) oppositeOriginalAddressLinks.head.status == oppositeOriginalAddressLinks.last.status else true
 
-    if (!oppositeStatusChange && (matchAddr && sameStatus && matchContinuity && administrativeClassesMatch && trackNotUpdated && originalTrackContinuous && (newLinks || oppositeTrackNotUpdated) && !(hasCalibrationPoint || hasParallelLinkOnCalibrationPoint)) &&
+    if (!oppositeStatusChange && (matchAddr && sameStatus && matchContinuity && administrativeClassesMatch && trackNotUpdated && originalTrackContinuous &&
+      (newLinks || oppositeTrackNotUpdated) && !(hasCalibrationPoint || hasParallelLinkOnCalibrationPoint)) &&
         administrativeClassesMatch && oppositeStatusNotChanged && originalAdministrativeClassContinuous) {
-      val pl1OriginalAddressSet = if (pl1.reversed)
-        pl1.copy(originalStartAddrMValue = pl2.originalStartAddrMValue, roadwayId = pl1.roadwayId)
-      else
-        pl1.copy(originalEndAddrMValue = pl2.originalEndAddrMValue, roadwayId = pl2.roadwayId)
+      val pl1OriginalAddressSet = {
+        if (pl1.reversed)
```

Comment:
Saaks nillittää nää
Math.max(pl1.originalEndAddrMValue, pl2.originalEndAddrMValue),
ja nää
Math.min(pl1.originalStartAddrMValue, pl2.originalStartAddrMValue)
muuttujiin tohon ennen tätä iffiä?

Reply:
... ja saisko originalStartAddrMValuet ennen endejä copyissa? (...vai mmmmiks ne on noin päin?). 😅

Reply:
Fixed